README.md

```diff
@@ -224,7 +224,7 @@ If needed, you can specify the Scala version with `-PscalaVersion=2.13`.
 There are two code quality analysis tools that we regularly run, spotbugs and checkstyle.
 
 #### Checkstyle ####
-Checkstyle enforces a consistent coding style in Kafka.
+Checkstyle enforces a consistent coding style in Kafka. Make sure you pass it before you submit.
```

Comment:
We should remind developers that `spotlessCheck` will fail if env is JDK8. `Make sure you pass it before you submit.` is a bit verbose since we have QA :)